### PR TITLE
[13.0][MIG] product_mrp_info

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Superseed https://github.com/OCA/manufacture/pull/419
CC @dreispt @andreampiovesana @sudhir-erpharbor @ForgeFlow

### CODE REVIEW

File: product_mrp_info/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* Lois Rilo <lois.rilo@eficent.com>
```
Comment (MiquelRForgeFlow) [inline product_mrp_info/readme/CONTRIBUTORS.rst]:
```suggestion
* Lois Rilo <lois.rilo@forgeflow.com>
```

File: product_mrp_info/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Product MRP Info",
+    "version": "13.0.1.0.0",
+    "development_status": "Beta",
+    "license": "LGPL-3",
+    "author": "Eficent, Odoo Community Association (OCA)",
```
Comment (MiquelRForgeFlow) [inline product_mrp_info/__manifest__.py]:
```suggestion
    "author": "ForgeFlow, Odoo Community Association (OCA)",
```

File: product_mrp_info/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
```
Comment (MiquelRForgeFlow) [inline product_mrp_info/__manifest__.py]:
```suggestion
# Copyright 2019 ForgeFlow S.L.
```

File: product_mrp_info/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 ForgeFlow S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Product MRP Info",
+    "version": "13.0.1.0.0",
+    "development_status": "Beta",
+    "license": "LGPL-3",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "maintainers": ["lreficent"],
```
Comment (MiquelRForgeFlow) [inline product_mrp_info/__manifest__.py]:
```suggestion
    "maintainers": ["LoisRForgeFlow"],
```

File: product_mrp_info/models/product.py [hunk 5]
```diff
@@ -0,0 +1,60 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
```
Comment (MiquelRForgeFlow) [inline product_mrp_info/models/product.py:1]: ...

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**